google_compute_engine/logger.py

```diff
@@ -18,7 +18,7 @@
 import logging
 import logging.handlers
```

Comment:
Logging is getting overridden - you should be able to exclusively use the logging module from compat.